src/concepts/components/nestedcomponents.md

```diff
@@ -1,3 +1,4 @@
+# Nested component
```

Comment:
I think it should say "nested components"

Reply:
I maintain the uppercase for "Nested..."